server/models/Conversation.js

```diff
@@ -0,0 +1,27 @@
+const mongoose = require("mongoose");
+
+const usersArrayBound = (val) => val.length <= 2;
+
+const conversationSchema = new mongoose.Schema(
+  {
+    users: {
+      type: [
+        {
+          type: mongoose.Schema.Types.ObjectId,
+          ref: "User",
+        },
+      ],
+      validate: [usersArrayBound, "Exceeds limit for number of users part of a conversation."],
+      required: true,
+      default: [],
+    },
+    lastMessage: {
```

Comment:
I noticed that we aren't using this field, should we remove it in this case or modify the controller to make use of it?

Reply:
I use it in the FE/IN Messaging PR. I'll reflect how I use it there here. Thanks for pointing this out.